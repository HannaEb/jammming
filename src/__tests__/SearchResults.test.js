import SearchResults from '../Components/SearchResults/SearchResults.js';
import renderer from 'react-test-renderer';

jest.mock('../Components/TrackList/TrackList.js', () => 'TrackList');

describe('SearchResults', () => {
    it('matches the snapshot', () => {
        const tree = renderer.create(<SearchResults />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
