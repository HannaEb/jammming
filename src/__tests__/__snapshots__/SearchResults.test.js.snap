// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResults matches the snapshot 1`] = `
<div
  className="SearchResults"
>
  <h2>
    Results
  </h2>
  <TrackList
    isRemoval={false}
  />
</div>
`;
